source/_integrations/androidtv.markdown

```diff
@@ -34,140 +36,25 @@ For Fire TV devices, the instructions are as follows:
-  required: false
-  type: string
-turn_off_command:
-  description: An ADB shell command that will override the default `turn_off` command.
-  required: false
-  type: string
-screencap:
-  description: Determines if album art should be pulled from what is shown onscreen.
-  required: false
-  default: true
-  type: boolean
-{% endconfiguration %}
-
-### Full Configuration
-
-```yaml
-# Example configuration.yaml entry
-media_player:
-  # Use the Python ADB implementation with a user-provided key to setup an
-  # Android TV device. Provide some app names and don't display other apps
-  # in the sources menu. Override the default turn on/off commands, and
-  # provide custom state detection rules.
-  - platform: androidtv
-    name: Android TV
-    device_class: androidtv
-    host: 192.168.0.222
-    adbkey: "/config/android/adbkey"
-    exclude_unnamed_apps: true
-    apps:
-      com.amazon.tv.launcher: "Fire TV"
-      some.background.app:  # this will never show up in the sources list
-      another.background.app: ""  # this will also never show up in the sources list
-    turn_on_command: "input keyevent 3"
-    turn_off_command: "input keyevent 223"
-    state_detection_rules:
-      'com.amazon.tv.launcher':
-        - 'standby'
-      'com.netflix.ninja':
-        - 'media_session_state'
-      'com.ellation.vrv':
-        - 'audio_state'
-      'com.plexapp.android':
-        - 'paused':
-            'media_session_state': 3  # this indentation is important!
-            'wake_lock_size': 1       # this indentation is important!
-        - 'playing':
-            'media_session_state': 3  # this indentation is important!
-        - 'standby'
-      'com.amazon.avod':
-        - 'playing':
-            'wake_lock_size': 4  # this indentation is important!
-        - 'playing':
-            'wake_lock_size': 3  # this indentation is important!
-        - 'paused':
-            'wake_lock_size': 2  # this indentation is important!
-        - 'paused':
-            'wake_lock_size': 1  # this indentation is important!
-        - 'standby'
-
-  # Use an ADB server to setup a Fire TV device and don't get the running apps.
-  - platform: androidtv
-    name: Fire TV
-    device_class: firetv
-    host: 192.168.0.222
-    adb_server_ip: 127.0.0.1
-    adb_server_port: 5037
-    get_sources: false
-```
+{% include integrations/config_flow.md %}
+
+{% include integrations/option_flow.md %}
+{% configuration_basic %}
+Configure Applications List:
+  description: Here you can define a list of applications where the keys are app IDs and the values are app names that will be displayed in the UI. If a name is not provided, the app will never be shown in the sources list. ([These app names](https://github.com/JeffLIrion/python-androidtv/blob/748d6b71cad611c624ef7526d9928431167531a3/androidtv/constants.py#L290-L308) are configured in the backend package and do not need to be included in your configuration).
```

Comment:
> If a name is not provided, the app will never be shown in the sources list.

That is only true if `exclude_unnamed_apps` (or its successor) is true. And I believe it is false by default.